### PR TITLE
stdout -> stderr for rtg

### PR DESCRIPTION
**Pull request by yunfeiguo**
- [ ] [RES-XXXX](https://binatechnologies.atlassian.net/browse/RES-XXXX)
- [ ] Blocking PRs: ADD HERE IF ANY
* People
  - [ ] Reviewers (>=1): ADD HERE
  - [ ] Merger (>=1): ADD HERE
  - [ ] FYI: ADD HERE IF ANY
- [ ] Release note (if needed)
- [ ] No code copied from outside Bina 
- [ ] Tests (Unit, Workflow, ITL & Gherkin) are passing
- [ ] PR labels, milestones & assignees

# Summary

suppress unwanted rtg output in final stdout.

### CODE REVIEW

File: compare_vcf.py [hunk 1]
```diff
@@ -191,9 +191,9 @@ def run(self):
         else:
             if self.log_to_file:
                 with utils.versatile_open(self.log_to_file, 'a') as logout:
-                    utils.run_shell_command(cmd, sys.stdout, logout)
+                    utils.run_shell_command(cmd, sys.stderr, logout)
             else:
-                utils.run_shell_command(cmd, sys.stdout, sys.stderr)
+                utils.run_shell_command(cmd, sys.stderr, sys.stderr)
```
Comment (marghoob) [inline compare_vcf.py]: @yunfeiguo why did you change stdout to stderr here?

Comment (yunfeiguo) [reply compare_vcf.py]: so both rtg and VarSim will print to stdout, but what rtg prints is only intermediate results. replacing `stdout` by `stderr` will make it possible to save the important stats by simple redirection `>`.
